cluster/src/test/scala/io/snappydata/cluster/QueryRoutingSingleNodeSuite.scala

```diff
@@ -239,8 +278,9 @@ class QueryRoutingSingleNodeSuite extends SnappyFunSuite with BeforeAndAfterAll
       // println("network server started")
       insertRows(tableName1, 1000, serverHostPort)
       insertRows(tableName2, 1000, serverHostPort)
-      query1(tableName1, tableName2, serverHostPort)
-      query2(tableName1, tableName2, serverHostPort)
+      // query1(tableName1, tableName2, serverHostPort)
+      (0 to 5).foreach(i => query2snc(tableName1, tableName2, serverHostPort, i))
+      // query2(tableName1, tableName2, serverHostPort)
```

Comment:
No. We just modified the test to run in loop and check the plan caching for proper working again and again.
The failure which was coming due to the pruning should  be a separate test and let us identify and pass it on to Shoubhik for testing.